### ISSUE TITLE
[Issue] The plugin does not work in Obsidian Desktop v1.7.4

### ISSUE THREAD

**Issue by lostwildland** [ivan-lednev/better-search-views#40]
The plugin does not work
Version:  macOS Obsidian Desktop v1.7.4

**Comment by Neeqstock** [comment]
Stopped working also for me, Windows version.

**Comment by chrsdk** [comment]
Doesn't work anymore on Android mobile with Obsidian 1.7.4

**Comment by hellspector** [comment]
Your can open the defalut search bar. type any words to search.  then switch to the page, better search view will work for the backlinks again .

it is a liitle annoying, but at the moment, it is the only way,  

hope it will be fixed.

**Comment by tdog123** [comment]
@hellspector - Your workaround worked for me! Thanks! 

Do you know if anyone is working on a fix for this? This plugin is a core feature of my setup.

Here's my use case: I'm working on 20 different projects. I have one long file with a chronological list of notes on all the projects I'm working on. Whenever I take a note, I parent tag the bullet with a link to the relevant Page (in my case Project). Then if I'm looking for info on the Project, I go to that Project page and I can see all the backlinks notes lined up nicely in chronological order at the bottom of the page.

I use other features like templates in Obsidian, so I'd _really_ like to stay in the tool and not use something else.

**Comment by ivan-lednev** [comment]
Hello, @tdog123,

> Do you know if anyone is working on a fix for this? This plugin is a core feature of my setup.
I'm aware of the issue, but since
1) I'm the only one working on this, and I'm doing this in my free time,
1) I'm also working on another plugin that has many more users,
2) there is a workaround,
this will have to wait.

Of course, I'll gladly review a PR if somebody submits it.

**Comment by tdog123** [comment]
@ivan-lednev - Thanks for making this plugin. I absolutely understand this is all done in your free time. Appreciate it greatly!

**Comment by bhopmann** [comment]
For rendering markdown in backlinks this plugin is working with Obsidian v1.7.0+: https://github.com/reply2za/obsidian-query-control

**Comment by hellspector** [comment]
> For rendering markdown in backlinks this plugin is working with Obsidian v1.7.0+: https://github.com/reply2za/obsidian-query-control

total different plugin.

**Comment by bhopmann** [comment]
> > For rendering markdown in backlinks this plugin is working with Obsidian v1.7.0+: https://github.com/reply2za/obsidian-query-control
> 
> total different plugin.

Yes, of course, but as I said: It works for rendering markdown in backlinks (that's what better-search-views plugin also does).

**Comment by astralspurs** [comment]
I would so appreciate a fix for this. My note setup is practically built around this Outliner style way of viewing notes and all their backlinks. I just want to make it known that there is enthusiasm around this issue - I understand, of course, @ivan-lednev has only limited time. The workaround does not seem to work for me anymore in Obsidian 1.7.7 (Flatpak Fedora Linux). Grateful for the plugin - best regards.

**Comment by ivan-lednev** [comment]
Fixed in 0.3.0. Please update and re-check.
